tekton/publish.yaml

```diff
@@ -80,7 +80,7 @@ spec:
       cp ${DOCKER_CONFIG} /workspace/docker-config.json
 
   - name: create-ko-yaml
-    image: golang:1.17.8
+    image: golang:1.18.3
```

Comment:
Any reason to update go here ? Mainly asking because the CI still uses `1.17`, so it might bring different behavior than on the CI 🙃

Reply:
No reason really, I just saw it was outdated and thought I'd fix it up. I don't mind taking it back to 1.17.8 though.